modules/xfem/src/dirackernels/XFEMPressure.C

```diff
@@ -8,69 +8,34 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "XFEMPressure.h"
-#include "Function.h"
-#include "GeometricSearchData.h"
-#include "ElementPairLocator.h"
-#include "FEProblem.h"
 
 registerMooseObject("XFEMApp", XFEMPressure);
 
 InputParameters
 XFEMPressure::validParams()
 {
-  InputParameters params = DiracKernel::validParams();
+  InputParameters params = XFEMIntegratedBC::validParams();
+  params.addClassDescription("Applies a pressure on an interface cut by XFEM.");
   params.addRequiredParam<unsigned int>("component", "The component for the pressure");
   params.addParam<Real>("factor", 1.0, "The magnitude to use in computing the pressure");
   params.addParam<FunctionName>("function", "The function that describes the pressure");
-  params.addClassDescription("Applies a pressure on an interface cut by XFEM.");
+  params.addParam<PostprocessorName>("postprocessor",
+                                     "Postprocessor that will supply the pressure value");
+  params.addParam<Real>("alpha", 0.0, "alpha parameter required for HHT time integration scheme");
+  params.set<bool>("use_displaced_mesh") = true;
```

Comment:
@jiangwen84 I also set the default to be displaced mesh.

Reply:
I guess the standard PressureBC sets `use_displaced_mesh = true`, so your change is reasonable.